plugins/flytekit-papermill/flytekitplugins/papermill/task.py

```diff
@@ -10,10 +10,12 @@
 from nbconvert import HTMLExporter
 
 from flytekit import FlyteContext, PythonInstanceTask
+from flytekit.configuration import SerializationSettings
 from flytekit.core.context_manager import ExecutionParameters
 from flytekit.deck.deck import Deck
 from flytekit.extend import Interface, TaskPlugins, TypeEngine
 from flytekit.loggers import logger
+from flytekit.models import task as _task_models
```

Comment:
can we not use the `_` anymore?  let's just say `task_models`

Reply:
Updated it.